pallets/phala/src/compute/base_pool.rs

```diff
@@ -1008,22 +906,30 @@ pub mod pallet {
 		/// Merges multiple nfts belong to one user in the pool.
 		///
 		/// TODO(mingxuan): should try to avoid 0 share nft mint.
```

Comment:
Comment "should try to avoid 0 share nft mint" can be changed.

Reply:
done